ansibleflow/venv.py

```diff
@@ -38,7 +39,9 @@ def execute_under_env(command, os_env=None):
         proc.stdin.write(long_cmd.encode('utf-8'))
         proc.stdin.flush()
 
-        proc.wait()
+        returncode = proc.wait()
+        if returncode:
+            sys.exit(returncode)
```

Comment:
Am I understanding this correcting, you're only interested in sys.exiting when the returncode is > 0? If you are, It might be good to be explicit with that.